specification/containerservice/resource-manager/Microsoft.ContainerService/stable/2021-02-01/managedClusters.json

```diff
@@ -3331,6 +3331,10 @@
           "type": "boolean",
           "description": "Whether the pod identity addon is enabled."
         },
+        "allowNetworkPluginKubenet": {
```

Comment:
Add or update exmaple for this field?

Reply:
will do after rebasing this pr: https://github.com/Azure/azure-rest-api-specs/pull/12311

Reply:
added, PTAL

preview link: https://review.docs.microsoft.com/en-us/rest/api/documentation-preview/managedclusters/createorupdate?view=azure-rest-preview&branch=result_openapiHub_production_1eeaa1429d4d#create-managed-cluster-with-podidentity-enabled